src/transformers/pipelines/automatic_speech_recognition.py

```diff
@@ -306,6 +320,20 @@ def _sanitize_parameters(
         if decoder_kwargs is not None:
             postprocess_params["decoder_kwargs"] = decoder_kwargs
         if return_timestamps is not None:
+            if self.type == "seq2seq" and return_timestamps:
+                raise ValueError("We cannot return_timestamps yet on non-CTC models apart from Whisper!")
+            if self.type == "ctc_with_lm" and return_timestamps != "word":
+                raise ValueError("CTC with LM can only predict word level timestamps, set `return_timestamps='word'`")
+            if self.type == "ctc" and return_timestamps not in ["char", "word"]:
```

Comment:
We could add behaviour where the user can pass the argument `return_timestamps=True` for CTC models, and we set it to `return_timestamps="word"` in the `_sanitize_parameters` method -> I was against this since it could lead to unexpected behaviour, so am instead in favour of the user passing  `return_timestamps="char"` or  `return_timestamps="word"` as required

Reply:
Yep agree with you on this!